WFD/RBD.py

```diff
@@ -43,9 +47,11 @@
 class RbdBot(WfdBot):
     """Bot to enrich/create info on Wikidata for RBD objects."""
 
-    def __init__(self, mappings, year, new=False, cutoff=None):
+    def __init__(self, mappings, year, new=False, cutoff=None,
+                 preview_file=None):
```

Comment:
The issue here is that I use `-preview_file` to control if a preview is generated rather than an upload. If I set a default then I would need to separate out the trigger mechanism into one more commandline argument.